sirepo/package_data/static/js/activait.js

```diff
@@ -1091,14 +1147,78 @@ SIREPO.app.directive('columnSelector', function(appState, activaitService, panel
     };
 });
 
+SIREPO.app.directive('imageViewer', function(appState, plotting) {
+    return {
+        scope: {
+            modelName: "@"
+        },
+        template: `
+          <div data-ng-if="imageInfo">
+              <div data-image-preview="imageInfo"></div>
+          </div>
+        `,
+        controller: function($scope) {
+            plotting.setTextOnlyReport($scope);
+            $scope.load = function(json) {
+                $scope.imageInfo = json.images;
+                $scope.imageInfo.method = "imagePreview";
+            };
+        },
+        link: function link(scope, element) {
+            plotting.linkPlot(scope, element);
+        },
+    };
+});
```

Comment:
I replaced the "imageViewer" view which uses the old stateless-compute style of loading images with a proper imageSamplesReport. This allows the server to cache the results. This will give you an example for the other items in #6825 which should replace all the image stateless-compute calls with Report or Animation models.

Reply:
I also replace the stateless-compute calls on the regression tab with Animation plots. @gurhar1133, could you update the plots on the comparison tab with similar changes?